cmd/kube-controller-manager/controller-manager.go

```diff
@@ -48,6 +48,7 @@ var (
 	cloudProvider   = flag.String("cloud_provider", "", "The provider for cloud services.  Empty string for no provider.")
 	cloudConfigFile = flag.String("cloud_config", "", "The path to the cloud provider configuration file.  Empty string for no configuration file.")
 	minionRegexp    = flag.String("minion_regexp", "", "If non empty, and -cloud_provider is specified, a regular expression for matching minion VMs.")
+	syncNodes       = flag.Bool("sync_nodes", true, "If true, and -cloud_provider/-machines is specified, sync nodes from the cloud provider/machine list. Default true.")
```

Comment:
Can you rename it to syncNodeList?  Soonish, node controller will also sync node status, kill pod on down minions, etc, that part if not optional afaict.